backend/db/migrations/20240328023836-create-spot.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
```

Comment:
you are missing the options object that defines the schema in your migration files.